tests/test_common.py

```diff
@@ -22,5 +25,16 @@ def test_normalize_interface(self):
         normalized = common.normalize_interface(('fd6d:404d:54cb::1', 9042))
         self.assertEqual(normalized, ('fd6d:404d:54cb:0:0:0:0:1', 9042))
 
+    @patch('ccmlib.common.is_win')
+    def test_is_modern_windows_install(self, mock_is_win):
```

Comment:
Could you add another set of assertions, for a version that is higher than 2.1? 3.12 or 3.0.9 should do